wd_capabilities.au3

```diff
@@ -14,9 +14,11 @@
 ; Language ......: English
 ; Description ...: A collection of functions used to dynamically build the Capabilities string required to create a WebDriver session
 ; Author ........: mLipok
-; Modified ......:
-; URL ...........:
-; Date ..........: 2022/04/07
+; Modified ......: Danp2
+; URL ...........: https://github.com/Danp2/au3WebDriver/blob/master/wd_capabilities.au3
+; URL ...........: https://w3c.github.io/webdriver/#capabilities
+; URL ...........: https://docs.yugabyte.com/preview/api/ysql/datatypes/type_json/primitive-and-compound-data-types
+; Date ..........: 2022/04/08
```

Comment:
should be changed before meging to master

Reply:
changed to current date in: [__WD_CapabilitiesNotation removed](https://github.com/Danp2/au3WebDriver/pull/285/commits/265f1f0e0e664324bfd34951557bc74a761678cc)